doc/tutorial/nan_tutorial.txt

```diff
@@ -0,0 +1,65 @@
+
+.. _nan_tutorial:
+
+=================
+Dealing with NaNs
+=================
+
+Having a model yielding NaNs or Infs is quite common if some of the tiny
+components in your model are not set properly. NaNs are hard to deal with
+because sometimes it is caused by a bug or error in the code, sometimes it's
+because of the numerical stability of your actual computing systerm, and even,
```

Comment:
I would say "your computational environment (library versions, etc.)"

On Mon, May 25, 2015 at 3:01 PM, Zhouhan LIN <notifications@github.com>
wrote:

> In doc/tutorial/nan_tutorial.txt
> <https://github.com/Theano/Theano/pull/2849#discussion_r30991436>:
>
> > @@ -0,0 +1,65 @@
> > +
> > +.. _nan_tutorial:
> > +
> > +=================
> > +Dealing with NaNs
> > +=================
> > +
> > +Having a model yielding NaNs or Infs is quite common if some of the tiny
> > +components in your model are not set properly. NaNs are hard to deal with
> > +because sometimes it is caused by a bug or error in the code, sometimes it's
> > +because of the numerical stability of your actual computing systerm, and even,
>
> It is the environmental configuration, like numpy, scipy, etc. I would put
> "your numpy version" instead.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Theano/Theano/pull/2849/files#r30991436>.
>